test/mocks/mockISelectionManagerTest.ts

```diff
@@ -0,0 +1,250 @@
+        it("should return false when there is no selection", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+
+            expect(selectionManager.containsSelection(selectionId)).toBeFalsy();
+        });
+
+        it("should return true when the selection is present", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+
+            selectionManager.select(selectionId);
+
+            expect(selectionManager.containsSelection(selectionId)).toBeTruthy();
+        });
+
+        it("should return false when the selection is not present", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+            const otherSelectionId: ISelectionId = createSelectionId("2");
+
+            selectionManager.select(selectionId);
+
+            expect(selectionManager.containsSelection(otherSelectionId)).toBeFalsy();
+        });
+    });
+
+    describe("select", () => {
+        it("should add a selection", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+
+            selectionManager.select(selectionId);
+
+            const selectedIds: ISelectionId[] = selectionManager.getSelectionIds();
+            expect(selectedIds).toHaveSize(1);
+            expect(selectedIds[0].equals(selectionId)).toBeTruthy();
+        });
+
+        it("should toggle a selection", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+
+            selectionManager.select(selectionId);
+            selectionManager.select(selectionId);
+
+            const selectedIds: ISelectionId[] = selectionManager.getSelectionIds();
+            expect(selectedIds).toHaveSize(0);
+        });
+
+        it("should replace a selection", () => {
+            const selectionId0: ISelectionId = createSelectionId("0");
+            const selectionId1: ISelectionId = createSelectionId("1");
+
+            selectionManager.select(selectionId0);
+            selectionManager.select(selectionId1);
+
+            const selectedIds: ISelectionId[] = selectionManager.getSelectionIds();
+            expect(selectedIds).toHaveSize(1);
+            expect(selectedIds[0].equals(selectionId1)).toBeTruthy();
+        });
+
+        it("should not update the selection when selection array is empty", () => {
+            const selectionId: ISelectionId = createSelectionId("1");
+
+            selectionManager.select(selectionId);
+            selectionManager.select([]);
+
+            const selectedIds: ISelectionId[] = selectionManager.getSelectionIds();
+            expect(selectedIds).toHaveSize(1);
+            expect(selectedIds[0].equals(selectionId)).toBeTruthy();
+        })
+
+        it("should add multiple selections at once", () => {
+            const selectionId0: ISelectionId = createSelectionId("0");
+            const selectionId1: ISelectionId = createSelectionId("1");
+            const selectionId2: ISelectionId = createSelectionId("2");
```

Comment:
You can create a variable with these selectionIds and then compare it with the result using `toEqual`

Reply:
Fixed.